src/util/hex.factory.js

```diff
@@ -71,7 +71,7 @@
         }
 
         function intToBigEndianValue(n, minBytes) {
-            n = BigInteger.valueOf(n);
```

Comment:
This will last unless someone tries to send 90,071,992.54740991 BTC, which is impossible..